pyproject.toml

```diff
@@ -18,6 +18,7 @@ classifiers = [
 ]
 dependencies = [
     "PyCryptodome",
```

Comment:
Thought we cut PyCryptodome? Anyway LGTM

Reply:
Let's cut it, but just in a different PR since CI takes too long (lol)

Reply:
See https://github.com/Future-House/paper-qa/pull/426